### PR TITLE
updting members.js file

### PR DESCRIPTION
**Pull request by mcheering**

### CODE REVIEW

File: public/js/members.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ $(document).ready(function () {
         console.log("you got some topics, bruh:", data)
         data.map(({ category }) => {
 
-          let newDiv = $("<div class='card'>")
+          let newDiv = $("<div class='card align-center col-10 studyTopic'>")
```
Comment (mcheering) [inline public/js/members.js:42]: adding styling in text

File: public/js/members.js [hunk 2]
```diff
@@ -40,4 +40,11 @@ $(document).ready(function () {
       }).catch((err) => console.error(err))
   }
   generateCard()
+
+  const studyTopic = $(".studyTopic").textContent
+
+  $(".studyTopic").on("click", (event) => {
```
Comment (mcheering) [inline public/js/members.js]: Starting the on click function for the topics

File: public/login.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
       <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
             integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
       <link rel="preconnect" href="https://fonts.gstatic.com">
-      <link href="../public/css/style.css" rel="stylesheet">
+      <link href="/stylesheets/style.css" rel="stylesheet">
```
Comment (mcheering) [inline public/login.html:13]: fixing routing

File: public/login.html [hunk 4]
```diff
@@ -42,9 +42,9 @@ <h2>Login</h2>
       </div>
 
       <!-- /container -->
-      <footer class=" footer container-fluid mt-auto py-3 bg-dark position-absolute bottom-0">
-            <div class="container mp"></div>
-            <p class="text-center"> copyright © Learn Something New App 2021</p>
+      <footer class=" footer container-fluid py-3 bg-dark">
+            <div class="container">
```
Comment (mcheering) [inline public/login.html:46]: removing extraneous text

File: public/login.html [hunk 5]
```diff
@@ -42,9 +42,9 @@ <h2>Login</h2>
...
+                  <p class="text-center"> copyright © Learn Something New App 2021</p>
```
Comment (mcheering) [inline public/login.html:47]: centering text for footer

File: public/members.html [hunk 6]
```diff
@@ -19,7 +19,7 @@
       <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
 
       <!-- Custom styles for this template -->
-      <link href="/css/style.css" rel="stylesheet">
+      <link href="/stylesheets/style.css" rel="stylesheet">
```
Comment (mcheering) [inline public/members.html:22]: Fixing routing

File: public/members.html [hunk 7]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
       </div>
       <div class="container">
             <h2>Quiz</h2>
+            <h4>Correct Terms</h4>
+            <ul id="correctAnswers">
+                  <!--This is where terms will go when they click the correct button-->
```
Comment (mcheering) [inline public/members.html:73]: Fixing formatting for where flashcard will go, and providing empty lists for when the user selects a term as correct or incorrect.

File: public/members.html [hunk 8]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+            </ul>
+            <h4>Incorrect Tems</h4>
+            <ul id="incorrectAnswers">
```
Comment (mcheering) [inline public/members.html:76]: Providing space for incorrect terms when user gets a word wrong.

File: public/members.html [hunk 9]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                  <!--This is where terms will go when they click the incorrect button-->
+            </ul>
```
Comment (mcheering) [inline public/members.html:78]: Providing comment so others know where generated html will go.

File: public/members.html [hunk 10]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
-                        <div class="scene scene--card position-relative">
-                              <div class="card" style="width:100%; height:100%;">
-                                    <div class="card__face card__face--front">
-                                          front
+                  <div class="flip-card mx-auto mb-5 my-5">
+                        <div class="flip-card-inner">
+                              <div class="flip-card-front">
+                                    <h4 id="cardTerm">Front</h4>
+                              </div>
+                              <div class="flip-card-back">
```
Comment (mcheering) [inline public/members.html:86]: Providing better solution for card to flip without javascript.

File: public/members.html [hunk 11]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                                    <div class="col">
+                                          <div class="row">
```
Comment (mcheering) [inline public/members.html:88]: Providing better solution for card to flip without javascript.

File: public/members.html [hunk 12]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                                                <p class="mt-5" id="termBack">back</p>
+                                          </div>
```
Comment (mcheering) [inline public/members.html:90]: backside of card where text will go.

File: public/login.html [hunk 13]
```diff
@@ -42,9 +42,9 @@ <h2>Login</h2>
       </div>
 
       <!-- /container -->
-      <footer class=" footer container-fluid mt-auto py-3 bg-dark position-absolute bottom-0">
-            <div class="container mp"></div>
-            <p class="text-center"> copyright © Learn Something New App 2021</p>
+      <footer class=" footer container-fluid py-3 bg-dark">
```
Comment (mcheering) [inline public/login.html:45]: .

Comment (mcheering) [inline public/login.html:45]: .

File: public/login.html (hunk 3 above)
Comment (mcheering) [inline public/login.html:13]: .

File: public/members.html [hunk 14]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
       </div>
       <div class="container">
             <h2>Quiz</h2>
+            <h4>Correct Terms</h4>
```
Comment (mcheering) [inline public/members.html:71]: Where correct words will go

File: public/members.html [hunk 15]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+            <ul id="correctAnswers">
```
Comment (mcheering) [inline public/members.html:72]: .

File: public/members.html (hunk 7 above)
Comment (mcheering) [inline public/members.html:73]: .

File: public/members.html [hunk 16]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                  <!--This is where terms will go when they click the correct button-->
+            </ul>
```
Comment (mcheering) [inline public/members.html:74]: .

File: public/members.html [hunk 17]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+            <h4>Incorrect Tems</h4>
```
Comment (mcheering) [inline public/members.html:75]: .

File: public/members.html [hunk 18]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+            <ul id="incorrectAnswers">
+                  <!--This is where terms will go when they click the incorrect button-->
```
Comment (mcheering) [inline public/members.html]: .

File: public/members.html [hunk 19]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
-                              <h4>Incorrect</h4>
-                              <ul id="incorrectAnswers">
-
-                              </ul>
-                        </div>
-                        <div class="scene scene--card position-relative">
-                              <div class="card" style="width:100%; height:100%;">
-                                    <div class="card__face card__face--front">
-                                          front
+                  <div class="flip-card mx-auto mb-5 my-5">
```
Comment (mcheering) [inline public/members.html:81]: .

File: public/members.html [hunk 20]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                        <div class="flip-card-inner">
```
Comment (mcheering) [inline public/members.html:82]: .

File: public/members.html [hunk 21]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                              <div class="flip-card-front">
```
Comment (mcheering) [inline public/members.html:83]: .

File: public/members.html [hunk 22]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                                    <h4 id="cardTerm">Front</h4>
```
Comment (mcheering) [inline public/members.html:84]: .

File: public/members.html [hunk 23]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                              </div>
```
Comment (mcheering) [inline public/members.html:85]: .

File: public/members.html [hunk 24]
```diff
@@ -68,42 +68,44 @@ <h2>Study Topics</h2>
...
+                              <div class="flip-card-back">
+                                    <div class="col">
```
Comment (mcheering) [inline public/members.html:87]: .

File: public/members.html (hunk 11 above)
Comment (mcheering) [inline public/members.html:88]: .

File: config/passport.js [hunk 25]
```diff
@@ -8,13 +8,13 @@ passport.use(new LocalStrategy(
   {
     usernameField: "email"
   },
-  function(email, password, done) {
+  function (email, password, done) {
```
Comment (mcheering) [inline config/passport.js:11]: .

File: config/passport.js [hunk 26]
```diff
@@ -8,13 +8,13 @@ passport.use(new LocalStrategy(
...
     // When a user tries to sign in this code runs
     db.User.findOne({
       where: {
         email: email
       }
-    }).then(function(dbUser) {
+    }).then(function (dbUser) {
```
Comment (mcheering) [inline config/passport.js:17]: .

File: config/passport.js [hunk 27]
```diff
@@ -36,11 +36,11 @@ passport.use(new LocalStrategy(
 // In order to help keep authentication state across HTTP requests,
 // Sequelize needs to serialize and deserialize the user
 // Just consider this part boilerplate needed to make it all work
-passport.serializeUser(function(user, cb) {
```
Comment (mcheering) [inline config/passport.js:39]: .

File: config/passport.js [hunk 28]
```diff
@@ -36,11 +36,11 @@ passport.use(new LocalStrategy(
...
+passport.serializeUser(function (user, cb) {
   cb(null, user);
 });
 
-passport.deserializeUser(function(obj, cb) {
```
Comment (mcheering) [inline config/passport.js:43]: .

File: config/passport.js [hunk 29]
```diff
@@ -36,11 +36,11 @@ passport.use(new LocalStrategy(
...
+passport.deserializeUser(function (obj, cb) {
```
Comment (mcheering) [inline config/passport.js:43]: .

File: config/passport.js [hunk 30]
```diff
@@ -36,11 +36,11 @@ passport.use(new LocalStrategy(
 // In order to help keep authentication state across HTTP requests,
 // Sequelize needs to serialize and deserialize the user
 // Just consider this part boilerplate needed to make it all work
-passport.serializeUser(function(user, cb) {
+passport.serializeUser(function (user, cb) {
```
Comment (mcheering) [inline config/passport.js:39]: .

File: package.json [hunk 31]
```diff
@@ -17,7 +17,7 @@
     "express": "^4.17.0",
     "express-session": "^1.16.1",
     "mysql2": "^1.6.5",
-    "passport": "^0.4.0",
+    "passport": "^0.4.1",
```
Comment (mcheering) [inline package.json:20]: .

File: public/js/members.js [hunk 32]
```diff
@@ -1,6 +1,15 @@
 $(document).ready(function () {
   // This file just does a GET request to figure out which user is logged in
   // and updates the HTML on the page
+
```
Comment (mcheering) [inline public/js/members.js:4]: .

File: public/js/members.js [hunk 33]
```diff
@@ -1,6 +1,15 @@
...
+  $('#logout').on("click", (event) => {
```
Comment (mcheering) [inline public/js/members.js:7]: .

File: public/js/members.js [hunk 34]
```diff
@@ -1,6 +1,15 @@
...
+    event.preventDefault()
```
Comment (mcheering) [inline public/js/members.js:8]: .

File: public/js/members.js [hunk 35]
```diff
@@ -1,6 +1,15 @@
...
+  })
```
Comment (mcheering) [inline public/js/members.js:9]: .

File: public/js/members.js [hunk 36]
```diff
@@ -1,6 +1,15 @@
...
+    $.get("/api/user_data").then(function (data) {
```
Comment (mcheering) [inline public/js/members.js]: .

File: public/js/members.js [hunk 37]
```diff
@@ -1,6 +1,15 @@
...
+      $(".member-name").text(data.email);
```
Comment (mcheering) [inline public/js/members.js]: .

File: public/js/members.js [hunk 38]
```diff
@@ -1,6 +1,15 @@
...
+    });
```
Comment (mcheering) [inline public/js/members.js]: .

File: public/js/members.js [hunk 39]
```diff
@@ -1,6 +1,15 @@
...
+
```
Comment (mcheering) [inline public/js/members.js:13]: .

File: public/js/members.js [hunk 40]
```diff
@@ -1,6 +1,15 @@
...
+
```
Comment (mcheering) [inline public/js/members.js:14]: .

File: public/js/members.js [hunk 41]
```diff
@@ -41,10 +50,10 @@ $(document).ready(function () {
   }
   generateCard()
 
-  const studyTopic = $(".studyTopic").textContent
 
-  $(".studyTopic").on("click", (event) => {
+
```
Comment (mcheering) [inline public/js/members.js:56]: .

File: public/js/members.js [hunk 42]
```diff
@@ -41,10 +50,10 @@ $(document).ready(function () {
...
+  $(".studyTopic").click(function (event) {
```
Comment (mcheering) [inline public/js/members.js]: .

File: public/js/members.js [hunk 43]
```diff
@@ -41,10 +50,10 @@ $(document).ready(function () {
...
     event.preventDefault()
-    console.log("button works")
+    console.log("you clicked a study topic")
```
Comment (mcheering) [inline public/js/members.js]: .

File: public/members.html [hunk 44]
```diff
@@ -26,7 +26,7 @@
       <nav class="navbar navbar-dark bg-dark">
             <div class="container-fluid position relative">
                   <a class="navbar-brand text-center" href="#">Flash Card App</a>
-                  <button class="btn btn-danger float-right position-absolute" href="/logout">
+                  <button id ="logout" class="btn btn-danger float-right position-absolute" href="/logout">
```
Comment (mcheering) [inline public/members.html:29]: .

File: public/members.html [hunk 45]
```diff
@@ -105,7 +105,7 @@ <h4 id="cardTerm">Front</h4>
       <!-- /container -->
       <footer class=" footer container-fluid py-3 bg-dark">
             <div class="container">
-                  <p class="text-center"> copyright © Learn Something New App 2021</p>
+                  <p class="text-center my-auto"> copyright © Learn Something New App 2021</p>
```
Comment (mcheering) [inline public/members.html:108]: .

File: public/stylesheets/style.css [hunk 46]
```diff
@@ -14,7 +14,7 @@ form.login {
     left: 0;
     bottom: 0;
     overflow: hidden;
-    margin-top: 3rem;
+    margin-top: 10rem;
```
Comment (mcheering) [inline public/stylesheets/style.css:17]: .

**Review by mcheering — COMMENTED** [review]
Updating repo to fix some styling and functionality.

**Review by ChrisBone84 — APPROVED** [review]
Matthew removed dups

**Pull request merged**